packages/react-devtools-core/src/backend.js

```diff
@@ -59,7 +61,7 @@ function connectToDevTools(options: ?ConnectOptions) {
   var messageListeners = [];
   var closeListeners = [];
   var uri = 'ws://' + host + ':' + port;
-  var ws = new window.WebSocket(uri);
+  var ws = websocket ? websocket : new window.WebSocket(uri);
   ws.onclose = handleClose;
   ws.onerror = handleClose;
   ws.onmessage = handleMessage;
```

Comment:
It feels odd that we slap these properties onto the passed object. Would it be better if this function took `send` and *returned* these handlers? Then the calling code could choose what to do with them (including putting them on a websocket or some completely different interface).